src/adapters/rubicon.js

```diff
@@ -325,9 +345,23 @@ var RubiconAdapter = function RubiconAdapter() {
         slots: slots,
         timeout: bidderRequest.timeout - (Date.now() - bidderRequest.auctionStart - TIMEOUT_BUFFER)
       };
-      var callback   = function () {
-        _bidsReady(slots);
-      };
+      var callback = function noop() {};
+
+      if(!_handleBidEvent(params => {
+
+        var slot = slots.find(slot => slot.getElementId() === params.elementId);
+        var ad = slot.getRawResponseBySizeId(params.sizeId);
+        var time = ((new Date).getTime() - _bidStart);
+
+        utils.logMessage(`Rubicon Project bid back for "${params.elementId}" size ${params.sizeId} at: ${time}`);
+
+        _makeBid(slot, ad);
+
+      })) {
+        callback = () => {
+          _bidsReady(slots);
+        }
```

Comment:
technically in Javascript semi-colons are not needed almost everywhere, but I'm generally a semi-colon user, just forgot here.

👍